src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,51 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab, TabsType } from '../../types/Tabs';
+
+export const Tabs: React.FC<TabsType> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const handleTabSelect = (tab: Tab) => {
+    if (tab.id !== selectedTabId) {
+      onTabSelected(tab);
+    }
+  };
+
+  const selectedTab = tabs
+    .find(({ id }) => id === selectedTabId) || tabs[0];
+
+  const selectedTabContent
+    = tabs.find(({ id }) => id === selectedTabId)?.content;
```

Comment:
Do you really need to find selectedTab one more time? You can use selectedTab variable